release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md

```diff
@@ -190,12 +182,20 @@ digraph "org/package:0.1.0" {
 }
 ```
 
+##### Compiler API
+
+- Introduced a set of builders in the Types API, which are used to construct complex types that have varying components
+
 ##### Ballerina Shell
 
 ##### Ballerina Update Tool
 
 #### Improvements
 
+##### Compiler API
+
+- Added semantic API support for the `client resource access` action
```

Comment:
```suggestion
- Added semantic API support for the client resource access action
```

Reply:
@MaryamZi - It is fine to highlight the action name within backticks to separate it out from the normal text. We have followed the same elsewhere also.

Reply:
We only emphasize keywords, variable names, etc. Which category does this come under?

Reply:
Action names are also like variable names and also this is done just for more clarity. Do we have a strong reason to not do it?

Reply:
Sorry, I was referring to code actions. We will not need backticks as this is a common action.

Reply:
Has been fixed.